ledger/src/blockstore.rs

```diff
@@ -3256,11 +3256,6 @@ impl Blockstore {
         Ok(())
     }
 
-    /// For tests
-    pub fn set_last_root(&mut self, root: Slot) {
```

Comment:
@CriesofCarrots - How do you feel about me ripping this (technically `pub`) `Blockstore` function out? It was added for a local cluster test, and this PR obsoletes it

Reply:
I don't mind ripping a test-only method out.